ui/app/templates/components/mfa-setup-step-two.hbs

```diff
@@ -0,0 +1,35 @@
+<p>
+  TOTP Multi-factor authentication (MFA) can be enabled here if it is required by your administrator. This will ensure that
+  you are not prevented from logging into Vault in the future, once MFA is fully enforced.
+</p>
+<div class="field has-top-margin-l">
+  <MessageError @errorMessage={{this.error}} class="has-top-margin-s" />
+  {{#if @warning}}
+    <AlertBanner @type="info" @title="MFA enabled" @message={{@warning}} class="has-top-margin-l" />
+  {{else}}
+    <div class="list-item-row">
+      <div class="center-display">
+        {{! template-lint-disable no-curly-component-invocation }}
+        {{qr-code text=@qrCode colorLight="#F7F7F7" width=155 height=155 correctLevel="L"}}
```

Comment:
Seems like this wouldn't have anything preventing us from using bracket invocation ?

Reply:
I don't either. I went ahead and updated.